core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java

```diff
@@ -1567,6 +1598,8 @@ private void validateDeleteUser(EntityStore store, UserEntity user1) throws IOEx
     Assertions.assertTrue(store.exists(user1.nameIdentifier(), Entity.EntityType.USER));
     Assertions.assertTrue(store.delete(user1.nameIdentifier(), Entity.EntityType.USER));
     Assertions.assertFalse(store.exists(user1.nameIdentifier(), Entity.EntityType.USER));
+    // delete again should return false
+    Assertions.assertFalse(store.delete(user1.nameIdentifier(), Entity.EntityType.USER));
```

Comment:
Do we need similar test cases for groups and roles?

Reply:
Yes, I think it is necessary, please let me update the pr.

Reply:
Done